### PR TITLE
17.10 takeover build

### LINKED ISSUE #2280: Build 17.10 takeover

### PR DESCRIPTION
**Pull request by Caleb-Ellis**
## Done

- Updated homepage with 17.10 takeover as per [design](https://app.zenhub.com/workspace/o/ubuntudesign/www.ubuntu.com-design/issues/94)
- Brief with content: https://docs.google.com/document/d/1NXayZgtgr7FGARzNymKcoufl72B8jEp_h6zjckprzk4/edit#

## QA

- Check that the homepage (www.ubuntu.com) has a new takeover that matches the brief/design
- Check both large and small screens
- Check that the links connect to the right urls

## Issue / Card

Closes #2280

### CODE REVIEW

**Comment by frankiegranato** [comment]
@Caleb-Ellis thank you for this. A few small issues:
"Brand new  ..." text is the wrong size. I think its supposed to be 20?  
Chevron needs to be corrected  
I know Spencer has supplied you new background and new laptop & screenshot assets to insert. So please let me know when they're in place. I'd like to review it again. 
Is the hover style on the Download button correct? @anthonydillon It seems a bit lackluster to me, but if thats the pattern then, its fine.

**Comment by Caleb-Ellis** [comment]
@frankiegranato I've made the corrections you suggested and added the right assets, so it should be good to go now. @anthonydillon I've used p-button--neutral for the download button but should I be using a different class for download buttons?

**Comment by frankiegranato** [comment]
Looks great from my perspective Caleb. One small design thing: as we discussed - perhaps the screenshot looks slightly fuzzy, so I think you’re going to look into resolving that by using the asset server. I'm going to add my +1 but lets make sure @spencerbygraves reviews this too.

**Comment by frankiegranato** [comment]
@Caleb-Ellis I meant to remind you to add the Google events to the links on the takeover :-)

**Comment by spencerbygraves** [comment]
@Caleb-Ellis I agree the image looks a little blurry, it would be great if you could look into that please. Thanks

**Comment by Caleb-Ellis** [comment]
@spencerbygraves I've just made the image of the laptop a higher res for large screens so that should help with the blurriness.

File: templates/takeovers/_1710_takeover.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <p class="intro-text">Brand new Ubuntu desktop!<br>
+        Same amazing experience!</p>
+        <p class="u-hidden u-visible--small">
+          <img class="laptop-image--small" src="{{ ASSET_SERVER_URL }}f9dcb391-Laptop.png?w=654" alt="" />
+        </p>
+        <p><a href="/download" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : '17.10 takeover', 'eventLabel' : 'Download 17.10 now', 'eventValue' : undefined });" class="p-button--neutral"><span>Download 17.10 now</span></a></p>
+        <p class="u-no-margin--bottom"><a href="/desktop/17.10" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Learn more', 'eventLabel' : 'Learn more about Ubuntu 17.10', 'eventValue' : undefined });" class="p-link--inverted">Learn more about Ubuntu 17.10&nbsp;›</a></p>
+      </div>
+      <div class="u-hidden--small col-8 u-vertically-center">
+        <img src="{{ ASSET_SERVER_URL }}f9dcb391-Laptop.png?w=654" alt="" />
```
Comment (deadlight) [inline templates/takeovers/_1710_takeover.html]: Can you add a descriptive alt tag here, please? Something like "Laptop running Ubuntu 17.10" or similar.

File: static/sass/takeovers/_1710-release.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
+@mixin p-takeover-1710-release {
+  .p-takeover--1710-release {
+    width: auto;
```
Comment (deadlight) [inline static/sass/takeovers/_1710-release.scss]: Sort CCS rules alphabetically. (The linter will catch this in future so we should keep it in mind)

**Pull request merged**